src/main.ts

```diff
@@ -508,17 +512,25 @@ export default class WhatsApp {
 
     this.keyPair = null;
 
-    this.qrPath = resolvePath(".", qrPath);
+    this.parameters = {...{
+      keysPath : "./keys.json",
+      qrPath : "./qrcode.png",
+      clientName : "WhatsApp forwarder",
+      clientShortName : "WhatsAppForwarder",
+      restoreSession : false
+    }, ...parameters}
```

Comment:
```suggestion
    this.opts = Object.assign({}, opts, this.opts);
```
It's more readable in my opinion and performs better (according to [this site](https://jsperf.com/comparing-object-asign-spread))